$OpenBSD: patch-libproxy_cmake_libproxy_cmk,v 1.3 2012/03/29 14:58:24 ajacoutot Exp $

Do not link with -lpthread (we use -pthread in CMAKE_C*_FLAGS)

--- libproxy/cmake/libproxy.cmk.orig	Mon Jun  6 23:44:37 2011
+++ libproxy/cmake/libproxy.cmk	Thu Mar 29 11:29:08 2012
@@ -2,7 +2,7 @@ add_library(libproxy SHARED extension_config.cpp exten
 if(WIN32)
   target_link_libraries(libproxy modman;ws2_32;${LIBPROXY_LIBRARIES})
 else()
-  target_link_libraries(libproxy modman;m;pthread;dl;${LIBPROXY_LIBRARIES})
+  target_link_libraries(libproxy modman;m;${LIBPROXY_LIBRARIES})
 endif()
 file(TO_NATIVE_PATH ${MODULE_INSTALL_DIR} MODULE_INSTALL_DIR)
 if(WIN32)
@@ -17,7 +17,7 @@ endif()
 
 set_property(SOURCE ${CMAKE_CURRENT_SOURCE_DIR}/proxy.cpp
              PROPERTY COMPILE_DEFINITIONS MODULEDIR="${MODULEDIR}";BUILTIN_MODULES=${BUILTIN_MODULES})
-set_target_properties(libproxy PROPERTIES PREFIX "" VERSION 1.0.0 SOVERSION 1)
+set_target_properties(libproxy PROPERTIES PREFIX "" VERSION ${LIBproxy_VERSION})
 set_target_properties(libproxy PROPERTIES LINK_INTERFACE_LIBRARIES "")
 install(TARGETS libproxy DESTINATION ${LIB_INSTALL_DIR})
 install(FILES   proxy.h  DESTINATION ${INCLUDE_INSTALL_DIR})
